packages/e2e/features/ui/components/authenticator/sign-up-with-email.feature

```diff
@@ -20,3 +20,13 @@ Feature: Sign Up with Email
     And I confirm the password "VALID_PASSWORD"
     And I click the "Create Account" button
     Then I see "Confirmation Code"
+
+@next @react @vue @angular
+  Scenario: Email field autocompletes username
+    And "Email" field autocompletes "username"
```

Comment:
TIL hat `username` is still preferred even if it's an email!

https://www.chromium.org/developers/design-documents/form-styles-that-chromium-understands

I figured for Sign In, it'd be `username`, but for Sign _Up_, it'd be `type="email"` and autocomplete `email` too!

Reply:
What do you think about adding a description (https://cucumber.io/docs/gherkin/reference/#descriptions) below the `Scenario` to describe this test & behavior?

> When signing up with an `Email`, `autocomplete` prefers usage of `username` instead of `email`.
> See: <some reference link>

Reply:
Will do!